cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet

```diff
@@ -0,0 +1,26 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'build-cop-target-low',
+        rules: [
+          {
+            alert: '%s-low' % job_name_regex,
+            expr: |||
+              sum(rate(prowjob_state_transitions{job="plank",job_name=~"%s",job_name!~"rehearse.*",state="success"}[30m]))/sum(rate(prowjob_state_transitions{job="plank",job_name=~"%s",job_name!~"rehearse.*",state=~"success|failure"}[30m])) < 1
+            ||| % [job_name_regex, job_name_regex],
+            'for': '5m',
+            labels: {
+              severity: 'slack',
+              team: 'build-cop',
+            },
+            annotations: {
+              message: '%s jobs have been below the target (100%%) for 5 minutes.' % job_name_regex,
+            },
+          }
+          for job_name_regex in ['branch-.*-images', 'release-.*-4.1', 'release-.*-4.2', 'release-.*-upgrade.*']
```

Comment:
Are these all actually 100%? Do we not have e2e-aws ones anymore?

Reply:
No ... nothing about e2e-aws any more. @bparees updated the gdoc and build-cop board already.

Reply:
the release jobs include e2e-aws execution.

Reply:
e.g. `release-openshift-ocp-installer-e2e-aws-4.1` which is picked up by `release-.*-4.1`